src/monument/__tests__/getMonuments.test.ts

```diff
@@ -0,0 +1,36 @@
+import getMonuments from "../getMonuments";
+import { type Monument } from "../types";
+
+describe("Given the getMonuments function", () => {
+  describe("When it's called", () => {
+    test("Then it should return a monuments list with Angkor Wat and Colosseum", async () => {
```

Comment:
¿En qué os basáis para decir que debería devolver estos dos?

Reply:
Porque poner una lista de monumentos no sería correcto no?

Reply:
¿La función `sum` debería devolver "la suma de los números"?

Reply:
No, pero debe de devolver una lista de monumentos que puede cambiar y el test entonces no pasará

Reply:
Efectivamente.